Ginger/GingerCore/Platforms/PlatformsInfo/JavaPlatform.cs

```diff
@@ -148,7 +148,15 @@ public override Act GetPlatformActionByElementInfo(ElementInfo elementInfo, Elem
             Act elementAction = null;
             if (elementInfo != null)
             {
-                List<ActUIElement.eElementAction> elementTypeOperations = GetPlatformUIElementActionsList(elementInfo.ElementTypeEnum);
+                List<ActUIElement.eElementAction> elementTypeOperations = new List<ActUIElement.eElementAction>();
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'elementTypeOperations'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5877667)